app/uk/gov/hmrc/tai/model/domain/EmploymentBuilder.scala

```diff
@@ -35,17 +35,17 @@ class EmploymentBuilder @Inject()(auditor: Auditor) {
 
     def associatedEmployment(account: AnnualAccount, employments: Seq[Employment], nino: Nino, taxYear: TaxYear)(
       implicit hc: HeaderCarrier): Option[Employment] =
-      employments.filter(_.employerDesignation == account.employerDesignation) match {
+      employments.filter(_.sequenceNumber == account.sequenceNumber) match {
         case Seq(single) =>
           logger.warn(s"single match found for $nino for $taxYear")
           Some(single.copy(annualAccounts = Seq(account)))
         case Nil =>
           logger.warn(s"no match found for $nino for $taxYear")
           auditAssociatedEmployment(account, employments, nino.nino, taxYear.twoDigitRange)
         case many =>
-          logger.warn(s"multiple matches found for $nino for $taxYear")
+          logger.error(s"multiple matches found for $nino for $taxYear")
```

Comment:
Can we turn this back to a warn for now and we need to check it in prod to see if we still need this.

Reply:
Changed to warn